### PR TITLE
Goerli Deployment

### PR DESCRIPTION
**Pull request by olvresc**
## Description of Changes

[Link to Jira Ticket](https://halodao.atlassian.net/browse/LEND-80)

### Developer Checklist:

- [x] I have followed the guidelines in our Contributing document
- [x] This PR has a corresponding JIRA ticket
- [x] My branch conforms with our naming convention i.e. `feature/HDEV-XXX-description`
- [x] I have written new tests for your core changes, as applicable
- [x] I have successfully ran tests locally
- [x] I have formatted my code using format document in VSCode

### Reviewers Checklist:

- [ ] Code is readable and understandable; any unclear parts have explanations
- [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
- [ ] I have successfully ran tests locally

### CODE REVIEW

**Comment by olvresc** [comment]
Deployment docs here: https://halodao.atlassian.net/wiki/spaces/XAVE/pages/230621185/Lending+Market+Goerli+Contracts

Pending: Adding `LP_XSGD_USDC` and `LP_EURS_USDC` tokens

**Comment by olvresc** [comment]
Deployed Price Oracle for `LP_XSGD_USDC` and `LP_EURS_USDC`. Documented here: https://halodao.atlassian.net/wiki/spaces/XAVE/pages/230621185/Lending+Market+Goerli+Contracts?focusedCommentId=230883329

PR for adding the `lpAssets` and `priceOracles` in `halodao-contract-addresses`: https://github.com/xave-finance/halodao-contract-addresses/pull/56

**Comment by olvresc** [comment]
LP assets added: https://halodao.atlassian.net/wiki/spaces/XAVE/pages/230621185/Lending+Market+Goerli+Contracts?focusedCommentId=230981633

File: markets/halo/index.ts [hunk 1]
```diff
@@ -92,6 +92,15 @@ export const HaloConfig: IHaloConfiguration = {
...
     },
     [eEthereumNetwork.tenderly]: {},
+    [eEthereumNetwork.goerli]: {
+      DAI: '0xEF8143f052d225162C2660a0C7954ea3c6CAbb65',
+      WETH: '0x87fb8C797709176f890A7FBDe6185B66cabC5B93',
+      USDC: '0x6edE2292C4e9d3151169525af68D300f0C4EDbe1',
+      USDT: '0xCEF6014D0834d922F99264aF64C5A51eFbE9735f',
+      WBTC: '0x175ef0f7accf7cb3DEC16185e320873fe3cc56Aa',
+      LP_XSGD_USDC: '0x5886F7BFCC5D61CD4Ad0DC34e72DEFa2c85dE104',
+      LP_FXPHP_USDC: '0x1fbA91Ce46d2453e2d461b1a0FcF76588a07fE41',
```
Comment (tracyarciaga) [inline markets/halo/index.ts:102]:
```suggestion
      LP_EURS_USDC: '0x1fbA91Ce46d2453e2d461b1a0FcF76588a07fE41',
```

**Review by tracyarciaga — APPROVED** [review]

**Comment by 0xAplki** [comment]
will close this pr since we are not deploying to goerli anymore

**Pull request closed without merging**